docs/uppy-core.mdx

```diff
@@ -91,7 +91,42 @@ The name of the file.
 
 #### `file.meta`
 
-Object containing file metadata. Any file metadata should be JSON-serializable.
+Object containing standard as well as user-defined metadata for each file. Any
+custom file metadata should be JSON-serializable. The following standard
+metadata will be stored on all file objects, but plugins may add more metadata.
+
+- `file.meta.name`
+  - Same as `file.name`.
+- `file.meta.type`
+  - Same as `file.type`.
+- `file.meta.relativePath`
+  - For any local folder that was drag-dropped or opened in Uppy, the files
+    inside the folder will have the `relativePath` metadata field set to their
+    path, relative to the folder. `relativePath` begins with the folder’s name
+    and ends with the file’s name. If opening or drag-dropping a file instead of
+    a folder, `relativePath` will be `null`. The same behaviour exists for
+    remote (provider) files, but the path will instead be relative to the user’s
+    selection (checkboxes). No leading or trailing slashes.
+  - **Local file example:** When drag-dropping a local folder `folder1` which
+    has a folder inside of it named `folder2` which has a file named `file`
+    inside of it, the `relativePath` meta field of the file will be
+    `folder1/folder2/file`. However if drag-dropping or opening `file` directly,
+    `relativePath` will be `null`.
+  - **Remote file example:** When checking the checkbox next to a remove
```

Comment:
Small typo: "next to a remoTe".

I think we should rewrite it to:

> Suppose we have a remote provider folder structure such as `/folder1/folder2/file`.
> Then, if the user checks the checkbox next to `folder1`, `file`'s `relativePath` will be `folder1/folder2/file`.
> However if the user first navigates into `folder1`, and only then checks the checkbox next to `folder2`, `relativePath` will be `folder2/file`.

Reply:
```suggestion
  - **Remote file example:** Suppose we have a remote provider folder structure such as `/folder1/folder2/file`.
    Then, if the user checks the checkbox next to `folder1`, `file`'s `relativePath` will be `"folder1/folder2/file"`.
    However if the user first navigates into `folder1`, and only then checks the checkbox next to `folder2`, `relativePath` will be `"folder2/file"`.
```